gst/nnstreamer/tensor_filter/README.md

```diff
@@ -3,13 +3,13 @@ title: tensor_filter
 ...
 
 # NNStreamer::tensor\_filter
-This is the main element of the whole NNStreamer project. This connects gstreamer data stream with neural network frameworks (NNFW) such as Tensorflow or Caffe.  
```

Comment:
The two spaces in the markdown file are for line separation.
Please keep this style.

Reply:
updated, plz review the changes again.